### PR TITLE
Update beatmap_ranking_criteria.md

### PR DESCRIPTION
**Pull request by XDerbyX**
edit #3
fixed some grammars again (again)

### CODE REVIEW

File: en/beatmap_ranking_criteria.md [hunk 1]
```diff
@@ -7,18 +7,18 @@ Frequently Asked Questions:
 
 -----------------------
 
-- **Q: Why my requested map is not yet ranked while it's already qualified the criteria?**
+- **Q: Why is my requested map isn't ranked yet while it already meets the qualification for the Ranking  Criteria?**
 - A: Maybe your requested map is not detected by system, you can request again via our Discord. Format: `[REQ ULANG](Link Beatmap)(mode)`.
```
Comment (troke12) [inline en/beatmap_ranking_criteria.md:11]: Ubah itu 
Please use this format : `[REQ](osu-beatmap-link)(mode)` **Only for Donors**

File: en/beatmap_ranking_criteria.md [hunk 2]
```diff
@@ -7,18 +7,18 @@ Frequently Asked Questions:
...
-- A: It might be happen when : it was fresh beatmap, wrong link, beatmap takeover system from osu! database is not yet updated.
+- **Q: Why can't i submit Beatmap Rank Request?**
+- A: It might happen when : it's a fresh beatmap, wrong link, or the beatmap takeover system from osu! database is not yet updated.
 
 -----------------------
 
-- **Q: How much time needed for my requested beatmap to be ranked?**
-- A: Depends on the Beatmap Nominator team, BAT need time to review and sort out Beatmaps before it can be ranked.
+- **Q: How much time is needed for my requested beatmap to be ranked?**
+- A: Depends on the Beatmap Nominator Team, BAT's needs time to review and sort out Beatmaps before it can be ranked.
```
Comment (troke12) [inline en/beatmap_ranking_criteria.md:22]: ubah BAT jadi BN

Comment (XDerbyX) [reply en/beatmap_ranking_criteria.md:22]: otw

File: en/beatmap_ranking_criteria.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ Frequently Asked Questions:
 
 
 #### Notes
-Before requesting, make sure the unranked mapset you're requesting :
+Before requesting, make sure the unranked mapset you're requesting is :
 
 - Graveyard, 1 month or more to avoid beatmap update (abandoned map).
```
Comment (troke12) [inline en/beatmap_ranking_criteria.md:35]: Graveyarded?

Comment (XDerbyX) [reply en/beatmap_ranking_criteria.md:35]: bisa bisa

**Pull request merged**